### DISCUSSION TITLE
Attack Squids

### DISCUSSION THREAD

**Discussion by EuanM199** [PurpurMC/Purpur#482] [Feature Requests]
I would like the ability for a config setting to turn on retaliate for glow squids and normal squids.
i don't know why but flying attack squids sound amazing

**Comment by RhythmicSys** [comment]
This could likely be done by seeing how the [Cows naturally agressive to players](https://github.com/PurpurMC/Purpur/blob/ver/1.19.3/patches/server/0222-Cows-naturally-aggressive-to-players-chance.patch) and [Dolphins naturally agressive to players](https://github.com/PurpurMC/Purpur/blob/ver/1.19.3/patches/server/0221-Dolphins-naturally-aggressive-to-players-chance.patch) patches were implemented, and using those as a reference
